client/templates/meetingseries/meetingSeriesEdit.js

```diff
@@ -135,6 +135,37 @@ Template.meetingSeriesEdit.events({
     // "show" event is fired shortly before BootStrap modal dialog will pop up
     // We fill the temp. client-side only user database for the user editor on this event
     'show.bs.modal #dlgEditMeetingSeries': function (evt, tmpl) {
+
```

Comment:
This code block (until line 168) appears four time with just minimal differences. Please think about a solution how you could provide a function which takes some parameters (look at the differences in detail) and generates the code block needed in the special case.

For example the parameter `tmplData` for the dialog is always nearly the same. You could generate it like this:
```
const generateTmplData = (element) => {
    return {
        isEditedBy: user.username,
        isEditedDate: formatDateISO8601Time(element.isEditedDate)
    }
}
```
For this block you would use it like this:
```
const tmplData = generateTmplData(tmpl.data);
```
For another block it would look like that:
```
const tmplData = generateTmplData(editItem._infoItemDoc);
```

If you take a close look at the other parts of this block I am sure you find further parts you can extract like shown in this simple example 😉